tests/strategies/test_collection_to_cuds_function.py

```diff
@@ -9,19 +9,22 @@
 from rdflib import URIRef
 
 from dlite_cuds.strategies.collection_to_cuds_function import CUDSFunctionStrategy
-from dlite_cuds.strategies.parse_collection import CollectionParseStrategy
+
+# from dlite_cuds.strategies.parse_collection import CollectionParseStrategy
```

Comment:
kan slettes

Reply:
In principle I agreed, the reason why I kept them now is because there tests are skipped, but need to be updated. This PR is part of that. I prefer  not to remove there yet because I prefer to keep the tests (as moved from otapi amiii) unchanged until we correct them for real. This is just so that we see everything from scratch.  The reason for the commented Iines now is the linters that should pass.

Reply:
ok, then it makes sense to keep